fl_compvis/README.md

```diff
@@ -0,0 +1,25 @@
+# Computer vision for the project:
+A service embeded in the project to acheive the computer vision part, it recieves request from the control/learning node and send the coordinates of the end effector (xe,ye) and the coordinates of the object (xo,yo).
+In this version we detect red object by color, and track the end effector after determine its zero position.
+
+ ## camera type:
+ Basler acA1300-200ue
+ 
+ ## Connection to camera:
+ camera connects to computer using USB-3.0 port and can't run on USB-2.0 port
+ we have two ways to grab images from basler camera:
+ * Using python library: [pyPylon](https://github.com/basler/pypylon)
+ * Using ROS packages: [pylon_camera](https://github.com/magazino/pylon_camera)
+                     and [camera_control_messages](https://github.com/magazino/camera_control_msgs)
+ and in both methods we have to install the Camera Software Suite for Linux.
+ 
+ ## Working guide:
+ * Follow the instruction in [Interfacing Basler camera to ROS](https://www.baslerweb.com/fp-1527759326/media/downloads/documents/application_notes/Interfacing_Basler_Cameras_with_ROS.pdf).
+ * After the first time follow that steps:
+    1. Run rosnode in terminal#1: 
```

Comment:
Better to have one launch file that started pylon_camera and fl_compvis ser.py at once. 
And maybe add that we have to run catkin_make to generate messages.

Reply:
Good idea, I will work on it.
the user will run catkin_make during the process of installing camera packages (steps in the pdf file) so i didn't mention that.